var/spack/repos/builtin/packages/bumpversion/package.py

```diff
@@ -34,4 +34,4 @@ class Bumpversion(PythonPackage):
     version('0.5.3', 'c66a3492eafcf5ad4b024be9fca29820')
     version('0.5.0', '222ba619283d6408ce1bfbb0b5b542f3')
 
-    depends_on('py-setuptools', type='build')
+    depends_on('py-setuptools', type=('build', 'run'))
```

Comment:
just to be sure: It wouldn't be a requirement in the final implementation to have every package have a run dependency on its build dependencies?

Reply:
I think in this case `bumpversion` is actually wrong, it fails at runtime if setuptools isn't available, so I think it's really not a simple `build` dependency.  Given that I was grabbing `link, run` deps in build_systems/python.py this one needed *something* that overlapped.  If/when I fix the bits in python.py this will change.